src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -307,6 +331,7 @@ export class OrganizationIntegrations extends AsyncComponent<
         status={integrations.length ? 'Installed' : 'Not Installed'}
         publishStatus="published"
         configurations={integrations.length}
+        categories={this.getCategories(provider.metadata.features)}
```

Comment:
Just inspecting element on sentry so I might not have the context, but why not just pass in the features from `provider.features`, it seems like it would remove the need for L294-L297 to reformat the feature names

Reply:
I think it would also remove the need for a `Set` to remove duplicates, but I'm not sure

Reply:
@leeandher It's because SentryApps don't actually have a list of features that are just basic strings so you'd need a different function to handle that.

Reply:
ahhh okay, sounds good